parent/pom.xml

```diff
@@ -692,8 +701,8 @@
 						<artifactId>tycho-packaging-plugin</artifactId>
 						<version>${tychoVersion}</version>
 						<configuration>
-							<format>'${BUILD_ALIAS}-v'yyyyMMdd-HHmm'-B${BUILD_NUMBER}'</format>
-							<archiveSite>true</archiveSite>
+							<jgit.dirtyWorkingTree>error</jgit.dirtyWorkingTree>
+							<format>'${BUILD_ALIAS}-v'yyyyMMdd-HHmm'-ci'</format>
```

Comment:
I get what CI means but why should that be exposed in our released versions ?

By your explanation and what I see in this patch does that then not *require* me to commit something before I can see if a change will have effect since any CI built content I have would be overruled by old builds ?